app/grandchallenge/workstations/static/workstations/js/session-control.mjs

```diff
@@ -55,10 +56,10 @@ function openWorkstationSession(element) {
                     // the open() already focuses the window
                     workstationWindow.focus();
                     removeSpinner(element);
-                };
+                }
 
                 potentialSessionOrigins.forEach((origin) => {
-                    sendSessionControlMessage(workstationWindow, origin, {loadQuery: query}, onMessageIsSuccess);
+                    sendSessionControlMessage(workstationWindow, origin, {loadPath: path, loadQuery: query}, onMessageIsSuccess);
```

Comment:
This means that we cannot merge this PR until CIRRUS 2023.05 is released, and CIRRUS 2023.04 is set as the previous version, right?

Reply:
Correct. The changes from https://github.com/DIAGNijmegen/rse-cirrus-core/pull/2415 need to be released in the new CIRRUS version before this PR can be merged and released.